tests/tests/fv3_ccpp_gsd_drag_suite

```diff
@@ -84,7 +84,7 @@ export DO_SAT_ADJ=.F.
 export LRADAR=.T.
 export LTAEROSOL=.T.
 
-export FV3_RUN=ccpp_gsd_run.IN
+export FV3_RUN=ccpp_gsd_run_drag_suite.IN
```

Comment:
No need to change this if we can add a switch in the existing `ccpp_gsd_run.IN`

Reply:
Thanks for making the change to `ccpp_gsd_run.IN`. I will revert this change when I pull your code into the latest version of gsd/develop (happening now).